AppDevKit/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.1.0</string>
+	<string>1.2.0</string>
```

Comment:
There's many update in demo but only a small fix in feature, I think 1.1.1 will be better.